django/test/testcases.py

```diff
@@ -303,7 +303,7 @@ def assertRedirects(self, response, expected_url, status_code=302,
                 path = urljoin(response.request['PATH_INFO'], path)
 
             if fetch_redirect_response:
-                if netloc:
+                if netloc not in ('', 'testserver'):
```

Comment:
Could you add a comment why we check on `'testserver'` and use the following syntax instead of `in tuple`, please. :smile: 

```python
if netloc and netloc != 'testserver':
    ...
```